main/unbound/APKBUILD

```diff
@@ -77,20 +78,29 @@ package() {
 
 libs() {
 	pkgdesc="unbound shared libraries"
+	depends="$depends_libs"
 
 	mkdir -p "$subpkgdir"/usr/lib
 	mv "$pkgdir"/usr/lib/lib*.so.* "$subpkgdir"/usr/lib/
 }
 
+openrc() {
+	depends="$depends_openrc"
```

Comment:
I don't understand why is necessary a custom openrc(). Also $depends_openrc is uninitialized/empty.

Reply:
> cleanup depends for subpackages

depends get inherited unless reset.

Reply:
Right even though so far many if not all openrc packages just inherit depends from main pkg.

I'd rather use 'depends=""'. Using an uninitialized variable that looks an official APKBUILD variable it's kinda misleading.

Reply:
It should not. In the future it will not. See https://github.com/alpinelinux/abuild/commit/801578a06a0ee602f3533bb0184c0b58b96e3fa1#diff-88af1a35b35ecf6afcb7687f5afc7bbc